bin/compile

```diff
@@ -1,31 +1,13 @@
-#!/usr/bin/env bash
-# bin/compile <build-dir> <cache-dir>
+#!/usr/bin/env ruby
 
-set -e
+# sync output
+$stdout.sync = true
 
-BUILD_DIR=$1
-CACHE_DIR=$2
+$:.unshift File.expand_path("../../lib", __FILE__)
+require "build_pack"
```

Comment:
is the unshift so the path is right to include the file? I would probably have used `require_relative` instead